ts/parts/Axis.ts

```diff
@@ -5794,11 +5794,11 @@ class Axis implements AxisComposition {
         var axis: Highcharts.Axis = this as any,
             options = this.options,
             tickAmount = options.tickAmount,
-            tickPixelInterval = options.tickPixelInterval;
+            tickPixelInterval = options.tickPixelInterval as number;
```

Comment:
We should leave this as `any` for now. Later on, we will fix typing for the options (using Partial and DeepPartial), so that this option is always a number.